Library/Formula/castxml.rb

```diff
@@ -0,0 +1,24 @@
+class Castxml < Formula
+  desc "C-family Abstract Syntax Tree XML Output"
+  homepage "https://github.com/CastXML/CastXML"
+  url "http://http.debian.net/debian/pool/main/c/castxml/castxml_0.1+git20160202.orig.tar.xz"
+  mirror "https://mirrors.ocf.berkeley.edu/debian/pool/main/c/castxml/castxml_0.1+git20160202.orig.tar.xz"
+  version "0.1+git20160202-1"
+  sha256 "8de10ad3e930a48a6602c3adf23ed27d3845de24c231d562ca5c23e6df8dd94a"
+
+  head "https://github.com/CastXML/castxml.git"
+
+  depends_on "cmake" => :build
+  depends_on "llvm" => "with-clang"
+
+  def install
+    mkdir "build" do
+      system "cmake", "..", *std_cmake_args
+      system "make", "install"
+    end
+  end
+
+  test do
+    system "castxml"
```

Comment:
Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!